### ISSUE TITLE
incorrect wording of "Parallel Jobs" setting

### ISSUE THREAD

**Issue by felixf4xu** [microsoft/vscode-cmake-tools#3209]
### Brief Issue Summary

In the setting of Parallel Jobs, it says:

>Cmake: Parallel Jobs
The number of parallel build jobs. Use zero to automatically detect the number of CPUs. Setting this to 1 will disable build parallelism.

When I test in Windows 11/ msbuild (vs2022 is also installed), 

1. When I set the value to 0, the cmake cmd is:
```
 ...\bin\cmake.EXE --build d:/github/asdf_Test/asdf/build --config Debug --target ALL_BUILD -j 18 --
```
it seems to be correct but due to a cmake msbuild issue (https://discourse.cmake.org/t/parallel-does-not-really-enable-parallel-compiles-with-msbuild/964), the "-j 18" does not work with msbuild.

2. When I set the value to 1, the cmake cmd is: (without `-j 18`)

```
 ...\bin\cmake.EXE --build d:/github/asdf_Test/asdf/build --config Debug --target ALL_BUILD --
```
On Win11 and msbuild, this cmd **actually** works and msbuild build the project in parallel!

### Summary of the wording of the setting:

>"Use zero to automatically detect the number of CPUs" 

is correct although cmake can't build in parallel (it's another issue); 

>"Setting this to 1 will disable build parallelism." 

may not be correct. Even from literal meaning, setting value of "1" (Value of `True` to c++ programmers) will disable the setting of "Parallel Jobs"? Very confusing and not the truth on win11/msbuild.

### Additional Information

related but not same issue: https://github.com/microsoft/vscode-cmake-tools/issues/1958

**Comment by benmcmorran** [comment]
Thanks for the feedback. Do you think this wording is accurate? We'll make the change if everything looks good.

> The number of parallel build jobs. Use zero to automatically detect the number of CPUs. Setting this to 1 will omit the parallelism flag (`-j`) from the underlying build command, which has a generator-dependent effect on build parallelism.

**Comment by felixf4xu** [comment]
OK to me.
